src/Compiler/Compiler.cpp

```diff
@@ -1539,10 +1539,20 @@ bool Compiler::RegisterCommands(TclInterpreter* interp, bool batchMode) {
         }
       }
 
+      // GTKWave sets its current directory to its bin dir for dependency
```

Comment:
Just in case this was not considered: GTKWave location will be different on user install.